cli/src/main/java/org/infinispan/cli/commands/Shutdown.java

```diff
@@ -42,7 +42,7 @@ public CommandResult exec(ContextAwareCommandInvocation invocation) {
    public static class Server extends CliCommand {
       public static final String CMD = "server";
 
-      @Arguments(completer = ServerCompleter.class)
+      @Arguments(description = "The list of server names", completer = ServerCompleter.class, required = true)
```

Comment:
```suggestion
      @Arguments(description = "The list of server names", completer = ServerCompleter.class)
```

Reply:
I would not run the command if there is no server name given and there are more servers, as this would mean shut down a cluster and loosing data, so 'cluster' option should be used

Reply:
`shutdown server` only stops the server to which the CLI is connected and not the cluster.